kratos/input_output/stl_io.cpp

```diff
@@ -52,6 +80,33 @@ void StlIO::ReadModelPart(ModelPart & rThisModelPart)
         ReadSolid(rThisModelPart);
 }
 
+void StlIO::WriteModelPart(ModelPart & rThisModelPart)
```

Comment:
```suggestion
void StlIO::WriteModelPart(const ModelPart & rThisModelPart)
```

the other version of this fct is deprecated